examples/tutorials/region-arguments.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
@willschlitzer You also need to rename this file to `regions.rst`

Reply:
Pushed.